doc/services/portability/posix/aep/index.rst

```diff
@@ -52,7 +52,7 @@ Minimal Realtime System Profile (PSE51)
     _POSIX_THREAD_CPUTIME, -1,
     _POSIX_THREAD_PRIO_INHERIT, 200809L, :kconfig:option:`CONFIG_PTHREAD_MUTEX`
     _POSIX_THREAD_PRIO_PROTECT, -1,
-    _POSIX_THREAD_PRIORITY_SCHEDULING, -1,
+    _POSIX_THREAD_PRIORITY_SCHEDULING, -1, :kconfig:option:`CONFIG_POSIX_SCHED`
```

Comment:
@golowanow, for now, since Zephyr does not support processes, we can say it's supported as far as providing a linkable symbol, but it's use falls under undefined behaviour. That's in the docs already, and it's a known issue.
```suggestion
    _POSIX_THREAD_PRIORITY_SCHEDULING, -1, :kconfig:option:`CONFIG_POSIX_SCHED` :ref:`†<posix_undefined_behaviour>`
```

Reply:
done